src/main/groovy/com/blackbaud/templates/tasks/RestProject.groovy

```diff
@@ -153,7 +162,23 @@ authorization.filter.enable=false
     }
 
     void createCrudResource(String resourceName, boolean addEntity, boolean addWireSpec) {
-        addResourceAndSupportingClasses(resourceName, "crud", addWireSpec)
+        addResourceAndSupportingClasses(resourceName, addWireSpec)
+
+        File resourceFile = basicProject.findFile("${resourceName}Resource.java")
+        FileUtils.appendAfterLine(resourceFile, "class", """\
+
+    @GetMapping("/{id}")
+    public ${resourceName} find(@PathVariable("id") UUID id) {
+        throw new IllegalStateException("implement");
+    }
+""")
+
+        File clientFile = basicProject.findFile("${resourceName}Client.java")
+        FileUtils.appendAfterLine(clientFile, "class", """\
+
+    @RequestLine("GET /{id}")
+    ${resourceName} find(@Param("id") UUID id);
+""")
```

Comment:
why aren't these two methods added to the template files?

Reply:
there's two tasks, one for crud and one for non-crud... the template is used for both

Reply:
there's not really a 'crud client' anymore so the crud one should probably be removed but i left it in so that there's at least a trivial example of how to implement a GET